src/shared/components/structureViewer/StructureViewerPanel.tsx

```diff
@@ -369,6 +373,11 @@ export default class StructureViewerPanel extends React.Component<IStructureView
                             style={{marginRight: 5, cursor: "pointer"}}
                         />
                         <i
+                            className={classnames("fa", {"fa-compress": this.isIncreasedSize, "fa-expand": !this.isIncreasedSize})}
```

Comment:
We may want to use different icons here instead of `fa-compress` and `fa-expand`